### PR TITLE
Gh279 feat lowercase foldername

### LINKED ISSUE #279: standardize on lowercase and bilingual folder and file names where possible
in order to avoid future issues/typos and to simplify and standardize we should standardize on lower case folder names.

also, folder names have to be bilingual to meet government of Canada requirements

### PR DESCRIPTION
**Pull request by uday-ssc**
closes #279

### CODE REVIEW

File: solutions/kcc-namespaces/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 This solution is a simple fork of the KCC Project Namespaces blueprint found at
 https://cloud.google.com/anthos-config-management/docs/tutorials/project-namespace-blueprint
 
-This simple solution will great a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespsace.
+This simple solution will create a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespsace.
```
Comment (davelanglois-ssc) [inline solutions/kcc-namespaces/README.md:7]: might as well fix the word namespace at the end of this line

File: solutions/landing-zone-v2/onboarding-tenant.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
-# Tenant Onboarding
```
Comment (davelanglois-ssc) [inline solutions/landing-zone-v2/onboarding-tenant.md:1]: rename this file to become onboarding-client.md

File: solutions/landing-zone-v2/onboarding-tenant.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+# client Onboarding
```
Comment (davelanglois-ssc) [inline solutions/landing-zone-v2/onboarding-tenant.md:1]: Title can use a capital "C" for Client

**Comment by davelanglois-ssc** [comment]
I would like to see all three MD files from this folder updated https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/gh279-feat-lowercase-foldername/solutions/landing-zone-v2

File: solutions/landing-zone-v2/onboarding-tenant.md (hunk 2 above)
Comment (uday-ssc) [reply solutions/landing-zone-v2/onboarding-tenant.md:1]: Done.

File: solutions/landing-zone-v2/onboarding-tenant.md (hunk 3 above)
Comment (uday-ssc) [reply solutions/landing-zone-v2/onboarding-tenant.md]: Done

File: solutions/kcc-namespaces/README.md (hunk 1 above)
Comment (uday-ssc) [reply solutions/kcc-namespaces/README.md]: Done

File: solutions/hierarchy/admin-experimentation/folder.yaml [hunk 4]
```diff
@@ -15,9 +15,9 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Folder
 metadata:
-  name: common-services-infrastructure
+  name: tests.admins.admin1 # kpt-set: tests.admins.${admin-name}
   namespace: hierarchy
 spec:
-  displayName: Common Services Infrastructure
+  displayName: admin1 # kpt-set: ${admin-displayname}
```
Comment (alaincormier-ssc) [inline solutions/hierarchy/admin-experimentation/folder.yaml:21]: since everything is now lowercase, the kpt-set can be:

`# kpt-set: ${admin-name}`

**Review by lucstjean-ssc — APPROVED** [review]
I second Alain's comment otherwise it /lgtm

File: solutions/gatekeeper-policies/naming-rules/project/setters.yaml [hunk 5]
```diff
@@ -26,4 +26,4 @@ data:
   # Please change the values below
   #
   client_code: ^(aa|bb|cc)
-  env_code: d # (d = dev, p = production, s = sandbox, u = uat)
\ No newline at end of file
+  env_code: d # (d = dev, p = production, s = experimentation, u = uat)
```
Comment (davelanglois-ssc) [inline solutions/gatekeeper-policies/naming-rules/project/setters.yaml:29]: @lucstjean-ssc same here

File: solutions/gatekeeper-policies/naming-rules/project/template.yaml [hunk 6]
```diff
@@ -84,7 +84,7 @@ spec:
           asset.kind == "Project"
           env_code = input.parameters.env_code
           not re_match(env_code, asset.metadata.name)
-          message := sprintf("Project '%s' does not match meet naming requirements: The env code is incorrect. It must be (d = dev, p = production, s = sandbox, u = uat).", [asset.metadata.name])
+          message := sprintf("Project '%s' does not match meet naming requirements: The env code is incorrect. It must be (d = dev, p = production, s = experimentation, u = uat).", [asset.metadata.name])
```
Comment (davelanglois-ssc) [inline solutions/gatekeeper-policies/naming-rules/project/template.yaml:87]: @lucstjean-ssc  and here

File: solutions/landing-zone-v2/onboarding-admin.md [hunk 7]
```diff
@@ -6,9 +6,9 @@
     ```
     cd <LZ_FOLDER>
     ```
-1. Get the hierarchy/admin-sandbox package
+1. Get the hierarchy/admin-experimentation package
     ```
-    kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/admin-sandbox@main ./landing-zone/hierarchy/Testing/Admins/<admin name>
+    kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/admin-experimentation@main ./landing-zone/hierarchy/Testing/Admins/<admin name>
```
Comment (davelanglois-ssc) [inline solutions/landing-zone-v2/onboarding-admin.md:11]: hierarchy/Testing/Admins needs to be hierarchy/tests/admins

**Review by Michael-McAuley-SSC — COMMENTED** [review]
Just a couple quick comments regarding plural(s)

File: solutions/gatekeeper-policies/naming-rules/project/README.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ This naming policy evaluates each project's name using a series of regular expre
 | Name | Description |
 | -------- | --------- |
 | client code | Two-character code representing a client. Multiple clients can be supported by using a regex that evaluates multiple codes such as this: '^(aa\|bb\|cc)' |
-| Environment code  | One character code representing the environment for the project: `d = dev, p = production, s = sandbox, u = uat`    |
+| Environment code  | One character code representing the environment for the project: `d = dev, p = production, s = experimentation, u = uat`    |
```
[the remark this answers is not in the document]
Comment (lucstjean-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/README.md:10]: No. I already approve the PR.

File: solutions/gatekeeper-policies/naming-rules/project/setters.yaml (hunk 5 above)
Comment (lucstjean-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/setters.yaml:29]: Already approved PR :)

File: solutions/hierarchy/core-experimentation/tests/Unittest/folder.yaml [hunk 9]
```diff
@@ -15,9 +15,9 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Folder
 metadata:
-  name: testing.unittest
+  name: tests.unittest
```
Comment (alaincormier-ssc) [inline solutions/hierarchy/core-experimentation/tests/Unittest/folder.yaml:18]: "unittest" is singular in the name, but plural in the display name

File: solutions/hierarchy/admin-experimentation/folder.yaml (hunk 4 above)
Comment (uday-ssc) [reply solutions/hierarchy/admin-experimentation/folder.yaml]: done

File: solutions/hierarchy/admin-experimentation/setters.yaml [hunk 10]
```diff
@@ -20,5 +20,5 @@ metadata:
     config.kubernetes.io/local-config: "true"
 data:
   admin-name: 'admin1' # lowercase only
-  admin-displayname: 'Admin1'
+  admin-displayname: 'admin1'
```
[the remark this answers is not in the document]
Comment (uday-ssc) [reply solutions/hierarchy/admin-experimentation/setters.yaml]: done

**Comment by uday-ssc** [comment]
> 

@davelanglois-ssc  yes there are three MD files in that folder , i think they all are there
image

File: solutions/hierarchy/core-env/audits/folder.yaml [hunk 11]
```diff
@@ -15,9 +15,9 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Folder
 metadata:
-  name: audit
+  name: audits
   namespace: hierarchy
 spec:
-  displayName: Audit
+  displayName: audits
```
[the remark this answers is not in the document]
Comment (uday-ssc) [reply solutions/hierarchy/core-env/audits/folder.yaml:21]: audits

File: solutions/hierarchy/core-experimentation/tests/Unittest/folder.yaml (hunk 9 above)
Comment (uday-ssc) [reply solutions/hierarchy/core-experimentation/tests/Unittest/folder.yaml]: fixed

File: solutions/gatekeeper-policies/naming-rules/project/template.yaml (hunk 6 above)
Comment (lucstjean-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/template.yaml:87]: This is ok with me too

File: solutions/gatekeeper-policies/naming-rules/project/README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (uday-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/README.md:10]: Resolving this

File: solutions/kcc-namespaces/README.md [hunk 12]
```diff
@@ -4,7 +4,7 @@
 This solution is a simple fork of the KCC Project Namespaces blueprint found at
 https://cloud.google.com/anthos-config-management/docs/tutorials/project-namespace-blueprint
 
-This simple solution will create a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespace.
+This simple solution will create a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespsace.
```
Comment (davelanglois-ssc) [inline solutions/kcc-namespaces/README.md:7]: namespace was reverted somehow,, please fix again

Comment (uday-ssc) [reply solutions/kcc-namespaces/README.md]: Ah that merging , to fix conflicts !!

Comment (uday-ssc) [reply solutions/kcc-namespaces/README.md]: done

File: solutions/landing-zone-v2/onboarding-admin.md (hunk 7 above)
Comment (uday-ssc) [reply solutions/landing-zone-v2/onboarding-admin.md]: done

**Review by lucstjean-ssc — DISMISSED** [review]
still /lgtm

**Comment by davelanglois-ssc** [comment]
@uday-ssc please update this image https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/gh279-feat-lowercase-foldername/solutions/landing-zone-v2/img/multi-org.png

**Comment by davelanglois-ssc** [comment]
@uday-ssc  would you mind replacing the word sandbox under this folder https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/gh279-feat-lowercase-foldername/solutions/project

**Comment by uday-ssc** [comment]
> @uday-ssc please update this image https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/gh279-feat-lowercase-foldername/solutions/landing-zone-v2/img/multi-org.png

done

**Comment by davelanglois-ssc** [comment]
please update this line to fit the experimentation code "e" in the project id
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/6f4e1825bc5f9b0c77080ab669f5af6c6699af47/solutions/project/project-experimentation/setters.yaml#L33

File: solutions/gatekeeper-policies/naming-rules/project/README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/README.md:10]: change environment code to e

File: solutions/gatekeeper-policies/naming-rules/project/setters.yaml (hunk 5 above)
Comment (davelanglois-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/setters.yaml:29]: change comments to e = experimentation

File: solutions/gatekeeper-policies/naming-rules/project/template.yaml (hunk 6 above)
Comment (davelanglois-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/template.yaml:87]: change comments to e = experimentation

File: solutions/gatekeeper-policies/naming-rules/project/README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (uday-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/README.md:10]: done

File: solutions/gatekeeper-policies/naming-rules/project/setters.yaml (hunk 5 above)
Comment (uday-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/setters.yaml:29]: done

File: solutions/gatekeeper-policies/naming-rules/project/template.yaml (hunk 6 above)
Comment (uday-ssc) [reply solutions/gatekeeper-policies/naming-rules/project/template.yaml:87]: done

**Comment by davelanglois-ssc** [comment]
> please update this line to fit the experimentation code "e" in the project id
> 
> https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/6f4e1825bc5f9b0c77080ab669f5af6c6699af47/solutions/project/project-experimentation/setters.yaml#L33

@uday-ssc this is not implemented

**Comment by uday-ssc** [comment]
> 



> > please update this line to fit the experimentation code "e" in the project id
> > https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/6f4e1825bc5f9b0c77080ab669f5af6c6699af47/solutions/project/project-experimentation/setters.yaml#L33
> 
> @uday-ssc this is not implemented

Fixed

**Review by davelanglois-ssc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]

**Review by fmichaelobrien — COMMENTED** [review]
Reviewing post 1730 today

**Review by cartyc — APPROVED** [review]
/lgtm

**Comment by fmichaelobrien** [comment]
LGTM

**Pull request merged**